### PR TITLE
Further improvments for minimal settings

### PR DESCRIPTION
**Pull request by MaxUlysse**
# nf-core/sarek pull request

Many thanks for contributing to nf-core/sarek!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/sarek branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/sarek)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,8 +25,10 @@ Piellorieppe is one of the main massif in the Sarek National Park.
 - [#175](https://github.com/nf-core/sarek/pull/175) - Add `Sentieon` documentation
 - [#176](https://github.com/nf-core/sarek/pull/176) - Add empty `custom` genome in `genomes.config` to allow genomes that are not in `AWS iGenomes`
 - [#179](https://github.com/nf-core/sarek/pull/179) - Add `FreeBayes` germline variant calling
-- [#180](https://github.com/nf-core/sarek/pull/180) - Now saving Mapped Bams (and creating TSV) in minimal setting
+- [#180](https://github.com/nf-core/sarek/pull/180) - Now saving Mapped BAMs (and creating TSV) in minimal setting
```
Comment (ggabernet) [inline CHANGELOG.md:28]: Just out of curiousity, what is the minimal setting?

Comment (MaxUlysse) [reply CHANGELOG.md:28]: Just a fasta file for the reference genome.
We have this user working on some Bees, so there is a reference genome, but nothing else, no know variants or such.
So the plan is actually to use sarek to do some variant calling to get some known variants (SNPs) from only duplicates marked files and then use these variants within sarek to be able to finally do some recalibration.

Comment (ggabernet) [reply CHANGELOG.md:28]: ah, ok I see!

Comment (MaxUlysse) [reply CHANGELOG.md:28]: cf https://gatkforums.broadinstitute.org/gatk/discussion/1247/what-should-i-use-as-known-variants-sites-for-running-tool-x

**Comment by MaxUlysse** [comment]
GitHub seems to have some issue, so I'll wait for the tests to be OK to merge

File: CHANGELOG.md (hunk 1 above)
Comment (ggabernet) [reply CHANGELOG.md:28]: yes, it looks like that...

**Pull request merged**